blog-posts/scoped-registry.md

```diff
@@ -0,0 +1,100 @@
+---
+path: "/blog/scoped-registry/"
+date: "2021-12-30"
+title: "NPM scoped registry and authentication"
+featuredImage: images/npm-scoped-registry.jpg
+attribution:
+    creator: Paul Esch-Laurent
+    source: https://unsplash.com/photos/oZMUrWFHOB4
+seoMetaText: create scoped packages and scoped registry and configure authentication via npmrc
+teaserText: What are scoped packages, scoped registry and configure authentication via npmrc
+author: Mark Altmann
+authorSummary: dev@satellytes, ❤️ for Wombats
+
+---
+## TL;DR
```

Comment:
I removed the TL;DR; and put it into a conclusion and connected it with the story which makes the story smoother. 
But in general I think it depends on what you focus and what the people want to see. Personally when I am on my way to find a solution for a problem I am very grateful for TL;DR; because I want a solution or some piece of code and not pieces scattered over a story I have to read. On the other side when I like to read the story I could go on with the story to read and if I learn something in the end even better. But this is really a personal opinion I think. :+1:

Reply:
I understand and thanks for the changes 🙌